apps/avifenc.c

```diff
@@ -229,7 +229,7 @@ static void syntaxLong(void)
     printf("    -g,--grid MxN                     : Encode a single-image grid AVIF with M cols & N rows. Either supply MxN identical W/H/D images, or a single\n");
     printf("                                        image that can be evenly split into the MxN grid and follow AVIF grid image restrictions. The grid will adopt\n");
     printf("                                        the color profile of the first image supplied.\n");
-    printf("    -c,--codec C                      : codec to use (choose from versions list below)\n");
+    printf("    -c,--codec C                      : Codec to use (choose from versions list below)\n");
```

Comment:
Why is the `--qgain-map` option only available when `AVIF_ENABLE_JPEG_GAIN_MAP_CONVERSION` is defined? It doesn't make sense.

Reply:
Because currently the only way to create an avif with a gain map using avifenc is to convert a jpeg file that has a gainmap. So if that is not supported, there is no point in having a gain map quality flag, although we could still add it even if it doesn't do anything...?